content/pipeline/docker/syntax/steps.md

```diff
@@ -139,7 +139,7 @@ Use the status condition to override the default runtime behavior and execute st
 
 See the Conditions article for additional details:
 
-{{< link "pipeline/docker/config/conditions" >}}
+{{< link "pipeline/conditions" >}}
```

Comment:
I think it should change to `pipeline/docker/syntx/conditions` so that it routes to https://docs.drone.io/pipeline/docker/syntax/conditions/